CHANGELOG.md

```diff
@@ -438,8 +422,6 @@ external plugins
 
 ## [0.1.0] - 2018-05-15
 
-The first tagged version.
```

Comment:
This should probably have some message - maybe it just needs `### Added` as the change type?